src/trig.c

```diff
@@ -178,3 +62,10 @@ twin_fixed_t twin_tan(twin_angle_t a)
         return 0;
     return ((s << 15) / c) << 1;
 }
+
+void twin_sincos(twin_angle_t a, twin_fixed_t *sin, twin_fixed_t *cos)
```

Comment:
Here is the proofread version of your text:
Don't do that. You don't have to invoke both `twin_sin` and `twin_cos` in the function body of `twin_sincos`. Instead, you should calculate 5th order polynomial approximation in `twin_sincos`, and then both `twin_sin` and `twin_cos` should just be wrappers for `twin_sincos`.